src/containers/UserProfile/AboutUser.js

```diff
@@ -11,25 +11,27 @@ class AboutUser extends Component {
     super(props);
     this.state = { about: this.props.about };
     this.validator = new SimpleReactValidator();
-    this.handleUserInput = this.handleUserInput.bind(this);
     this.notificationDOMRef = React.createRef();
-    this.submitForm = this.submitForm.bind(this);
-    this.onCancel = this.onCancel.bind(this);
-    this.modalRef = React.createRef();
+    this.backdropRef = React.createRef();
     this.formRef = null;
   }
 
-  handleUserInput(e) {
+  handleUserInput = e => {
     const name = e.target.name;
     const value = e.target.value;
     this.setState({ [name]: value.trim() });
-  }
+  };
 
-  onCancel() {
+  onCancel = () => {
     this.formRef.reset();
-  }
+  };
+
+  removeBackdrop = () => {
+    var parent = document.getElementsByClassName("modal-backdrop fade show");
```

Comment:
@Humnaa rename `parent` to `element`

Reply:
@Humnaa also add a check here i.e. `if(element && element.length)`

Reply:
Renamed parent to element
and implement condition